include/zephyr/drivers/sensor.h

```diff
@@ -186,6 +186,10 @@ enum sensor_channel {
 	SENSOR_CHAN_GAUGE_DESIRED_VOLTAGE,
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
+	/** Open load detected, boolean with one bit per output */
```

Comment:
These should be in a private header file

Reply:
I actually have another PR (https://github.com/zephyrproject-rtos/zephyr/pull/67030) lined up which uses the same sensor channels. Should it then still be private, but duplicated?

Reply:
Yes, I think they should be private but duplicated

Reply:
done